### PR TITLE
Adding README and removing logs from index.js file

### PR DESCRIPTION
**Pull request by saikiranchalla1**
Removing unnecessary log file because they were redundant.
Adding documentation using README file

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
-console.log("Hello World!");
+console.log("Deleted unnecessary log statements");
 
-console.log("Hello Mars");
 
-console.log("Armando adding a new logger");
+var x  = 10;
+
+function add () {
```
Comment (arodriguez1176) [inline index.js]: Add should take input instead of returning 1 + 1

**Review by arodriguez1176 — CHANGES_REQUESTED** [review]
Modify the add mehtod, rest looks good to me

File: index.js (hunk 1 above)
Comment (saikiranchalla1) [reply index.js]: okay, I missed it. Fixing it now!

**Review by arodriguez1176 — APPROVED** [review]
LGTM! Thank you, @saikiranchalla1

**Pull request merged**